JoyboyCommunity/src/context/Theme.tsx

```diff
@@ -1,16 +1,31 @@
-import {createContext, useMemo} from 'react';
-import {useColorScheme} from 'react-native';
+import {createContext, useMemo, useState} from 'react';
+import {ColorSchemeName, useColorScheme} from 'react-native';
 
 import {DarkTheme, LightTheme, Theme} from '../styles';
 
-export const ThemeContext = createContext<Theme>(LightTheme);
+export type IThemeContext = {
+  theme: Theme;
+  toggleTheme: () => void;
+  scheme: 'dark' | 'light';
+};
```

Comment:
Not required but it would be better to use `ThemeContextType` as the name, since every other context uses the same naming.